### PR TITLE
Replace user with supplier

### PR DESCRIPTION
**Pull request by konradko**

### CODE REVIEW

File: company/views.py [hunk 1]
```diff
@@ -20,8 +21,8 @@ class CompanyRetrieveUpdateAPIView(generics.RetrieveUpdateAPIView):
 
     def get_object(self):
         return generics.get_object_or_404(
-            models.Company, users__sso_id=self.kwargs['sso_id']
-        )
+            Supplier, sso_id=self.kwargs['sso_id']
+        ).company
```
Comment (richtier) [inline company/views.py]: not a fan of this change as it turns one lookup into 2. `get_object_or_404` is  not evaluated lazily.

Comment (konradko) [inline company/views.py]: true, non-optimal. Will make another PR to fix
